### PR TITLE
Fix for RPMs for moderncppkafka.sh

### PR DESCRIPTION
**Pull request by Barthelemy**

### CODE REVIEW

File: moderncppkafka.sh [hunk 1]
```diff
@@ -5,12 +5,15 @@ requires:
...
+# this is header only library, so we can just copy it                                                                                                                                                    
 mkdir -p "$INSTALLROOT"
 cp -r "$SOURCEDIR/include" "$INSTALLROOT/include"
+rm "$INSTALLROOT/include/CMakeLists.txt" # for some reason it is there                                                                                                                                   
 
-# Modulefile
+# Modulefile                                                                                                                                                                                             
 mkdir -p "etc/modulefiles"
 alibuild-generate-module --lib > "etc/modulefiles/$PKGNAME"
+mkdir -p $INSTALLROOT/etc/modulefiles && rsync -a --delete etc/modulefiles/ $INSTALLROOT/etc/modulefiles
```
Comment (justonedev1) [inline moderncppkafka.sh:18]: why is the result of alibuild-generate-module copied to etc/modulefiles instead of directly into the $INSTALLROOT?

Comment (ktf) [reply moderncppkafka.sh:18]: In the past, if you had an incremental build, you might have ended up with stray bits in $INSTALLROOT which the rsync would have delete. Now we can probably simply remove the folder and regenerate them always, including in the incremental recipe.

Comment (justonedev1) [reply moderncppkafka.sh:18]: ah, okay thank you!

**Review by justonedev1 — APPROVED** [review]

File: moderncppkafka.sh (hunk 1 above)
Comment (ktf) [reply moderncppkafka.sh:18]: regenerate them with `alibuild-generate-module`, I mean.

Comment (ktf) [inline moderncppkafka.sh:18]:
```suggestion
mkdir -p "$INSTALLROOT/etc/modulefiles" && rsync -a --delete etc/modulefiles/ "$INSTALLROOT/etc/modulefiles"
```

**Pull request merged**